bridge/examples/mlflow/docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: '3.2'
+services:
+  db:
+    restart: always
+    image: mysql/mysql-server:5.7.28
+    container_name: mlflow_db
+    expose:
+    - "3306"
+    environment:
+    - MYSQL_DATABASE=mlflow
+    - MYSQL_USER=user
+    - MYSQL_PASSWORD=pass
+    - MYSQL_ROOT_PASSWORD=rootpass
+    volumes:
+    - ./dbdata:/var/lib/mysql
+    networks:
+    - A
```

Comment:
can we use a more descriptive network name so people know what they're looking at if they list the networks on their machine? Or is it namespaced with a more logical prefix automatically?

Reply:
The actual network used is the default bridge network.

Reply:
Changed to `mlflow_demo`.